Pipfile

```diff
@@ -4,14 +4,13 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
-python-hcl2 = "*"
```

Comment:
??

Reply:
> 2\. Remove duplicate dependencies (if they're in packages, no need to put them in `dev-packages`) + ran cleanup using `pipenv clean`

It's already in `packages`, no need to list it twice